### ISSUE TITLE
Music assistant does not cope well with HassPlayers that do not support stop

### ISSUE THREAD

**Issue by Rudd-O** [music-assistant/hass-music-assistant#2997]
### What version of Music Assistant has the issue?

2.3.0b29

### What version of the Home Assistant Integration have you got installed?

2024.9.1

### Have you tried everything in the Troubleshooting FAQ and reviewed the Open and Closed Issues and Discussions to resolve this yourself?

- [X] Yes

### The problem

Start play (clear queue) causes this nasty traceback, which it looks like it interferes with the clear queue functionality.

The player is playing in Queue Flow mode.

### How to reproduce

Add a HassPlayer from Home Assistant, which does not support the `media_player.stop` call.  Then play (clear queue) an album to it.  In short order, the traceback happens.

### Music Providers

Local.

### Player Providers

HASS.

### Full log output

```
Oct 02 15:24:12 penny.dragonfear music-assistant[3911036]: 2024-10-02 15:24:12.606 ERROR (MainThread) [music_assistant] Error doing task: Task exception was never retrieved
Oct 02 15:24:12 penny.dragonfear music-assistant[3911036]: Traceback (most recent call last):
Oct 02 15:24:12 penny.dragonfear music-assistant[3911036]:   File "/app/venv/lib/python3.12/site-packages/music_assistant/server/controllers/players.py", line 95, in wrapper
Oct 02 15:24:12 penny.dragonfear music-assistant[3911036]:     await func(self, *args, **kwargs)
Oct 02 15:24:12 penny.dragonfear music-assistant[3911036]:   File "/app/venv/lib/python3.12/site-packages/music_assistant/server/controllers/players.py", line 196, in cmd_stop
Oct 02 15:24:12 penny.dragonfear music-assistant[3911036]:     await player_provider.cmd_stop(player_id)
Oct 02 15:24:12 penny.dragonfear music-assistant[3911036]:   File "/app/venv/lib/python3.12/site-packages/music_assistant/server/providers/hass_players/__init__.py", line 225, in cmd_stop
Oct 02 15:24:12 penny.dragonfear music-assistant[3911036]:     await self.hass_prov.hass.call_service(
Oct 02 15:24:12 penny.dragonfear music-assistant[3911036]:   File "/app/venv/lib/python3.12/site-packages/hass_client/client.py", line 184, in call_service
Oct 02 15:24:12 penny.dragonfear music-assistant[3911036]:     return await self.send_command("call_service", **params)
Oct 02 15:24:12 penny.dragonfear music-assistant[3911036]:            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
Oct 02 15:24:12 penny.dragonfear music-assistant[3911036]:   File "/app/venv/lib/python3.12/site-packages/hass_client/client.py", line 225, in send_command
Oct 02 15:24:12 penny.dragonfear music-assistant[3911036]:     return await future
Oct 02 15:24:12 penny.dragonfear music-assistant[3911036]:            ^^^^^^^^^^^^
Oct 02 15:24:12 penny.dragonfear music-assistant[3911036]: hass_client.exceptions.FailedCommand: Entity media_player.lidia_s_sound_machine does not support this service.
Oct 02 15:24:12 penny.dragonfear music-assistant[3911036]: 
Oct 02 15:24:12 penny.dragonfear music-assistant[3911036]: The above exception was the direct cause of the following exception:
Oct 02 15:24:12 penny.dragonfear music-assistant[3911036]: 
Oct 02 15:24:12 penny.dragonfear music-assistant[3911036]: Traceback (most recent call last):
Oct 02 15:24:12 penny.dragonfear music-assistant[3911036]:   File "/app/venv/lib/python3.12/site-packages/music_assistant/server/controllers/player_queues.py", line 602, in stop
Oct 02 15:24:12 penny.dragonfear music-assistant[3911036]:     await self.mass.players.cmd_stop(queue_id, skip_forward=True)
Oct 02 15:24:12 penny.dragonfear music-assistant[3911036]:   File "/app/venv/lib/python3.12/site-packages/music_assistant/server/controllers/players.py", line 97, in wrapper
Oct 02 15:24:12 penny.dragonfear music-assistant[3911036]:     raise PlayerCommandFailed(str(err)) from err
Oct 02 15:24:12 penny.dragonfear music-assistant[3911036]: music_assistant.common.models.errors.PlayerCommandFailed: Entity media_player.lidia_s_sound_machine does not support this service.
Oct 02 15:24:14 penny.dragonfear music-assistant[3911036]: 2024-10-02 15:24:14.113 INFO (MainThread) [music_assistant.streams] Start Queue Flow stream for Queue Lidia's sound machine (via Music Assistant) - crossfade: False
```

### What version of Home Assistant Core are your running

2024.9.0

### What type of installation are you running?

Home Assistant Core

### On what type of hardware are you running?

Linux

**Comment by OzGav** [comment]
@Rudd-O can you check if this is fixed in 2.3.1 when it comes out soon

**Comment by OzGav** [comment]
We will close this soon assuming fixed
